auslib/web/admin/swagger/api.yaml

```diff
@@ -1199,37 +1199,37 @@ paths:
                 uniqueItems: true
                 minItems: 0
                 items:
-                  type: string
-                example: ["balrogadmin"]
+                  type: object
+
 
-  /users/roles:
+  /roles:
```

Comment:
I'm not sure I understand why you've modified this endpoint. It appears to return exactly the same data as /users, except in a different format. Is there a reason you're not able to build this UI with your new version of /users (that include the Roles a user holds), and the existing /users/roles endpoint?

Reply:
I modified it for consistency sake, so there would only be **/users and /roles**
Given your recommendation, we shall have one endpoint users that includes the roles and that is used to build the UI.